orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
```

Comment:
Sure! Basically, when you hit the tab key to indent a line of code, you can set your editor to use either a tab or any number of spaces (usually two). The only real difference is that tabs will make lines of code look more indented than code that uses spaces. Most people who use Ruby prefer spaces, but it's totally just a style preference that people have strong opinions about. You can probably just update a setting in your editor's preferences to switch to spaces if you want to go that route. Hope that's a little clearer!